MODULE.bazel

```diff
@@ -1,6 +1,146 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "rules_scala_annex")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+
+# TODO: Use buildifier instead of buildifier_prebuilt once
+# https://github.com/bazelbuild/bazel-central-registry/issues/380 is resolved
```

Comment:
I realized after I wrote this that we could just use `archive_override`, but I don't think it makes much sense to have users of the ruleset fetch an entire Go toolchain and compile buildifier from scratch when it's only used internally.